src/untyped.ts

```diff
@@ -18,8 +20,12 @@ module.exports = {
     'no-unbound-method': false,
     // Warns when a namespace qualifier (A.x) is
     'no-unnecessary-qualifier': false,
+    // Warns if a type assertion does not change the type of an expression.
```

Comment:
[`no-use-before-declare`](https://github.com/Shopify/tslint-config-shopify/pull/68/files#diff-b680e6509a051d5276efb842d9885773R30) and [`no-boolean-literal-compare`](https://github.com/Shopify/tslint-config-shopify/pull/68/files#diff-b680e6509a051d5276efb842d9885773R12) are already in the `untyped.ts` file?